trace/opencensus/trace/ext/django/config.py

```diff
@@ -19,7 +19,9 @@
 DEFAULT_DJANGO_TRACER_CONFIG = {
     'TRACER': 'opencensus.trace.tracer.context_tracer.ContextTracer',
     'SAMPLER': 'opencensus.trace.samplers.always_on.AlwaysOnSampler',
-    'REPORTER': 'opencensus.trace.reporters.print_reporter.PrintReporter'
+    'REPORTER': 'opencensus.trace.reporters.print_reporter.PrintReporter',
+    'PROPAGATOR': 'opencensus.trace.propagation.google_cloud_format.'
+                  'GoogleCloudFormatPropagator',
```

Comment:
Can you create a "glossary of terms" in some central place, with a short, clear description of each of these things?

Reply:
Done. I added the description in README file.